synapse/lib/hiveauth.py

```diff
@@ -6,9 +6,21 @@
 import synapse.lib.base as s_base
 import synapse.lib.cache as s_cache
 import synapse.lib.nexus as s_nexus
+import synapse.lib.config as s_config
 
 logger = logging.getLogger(__name__)
 
+reqValidRules = s_config.getJsValidator({
+    'type': 'array',
+    'items': {
+        'type': 'array',
+        'items': [
+            {'type': 'boolean'},
+            {'type': 'array', 'items': {'type': 'string'}},
+        ],
```

Comment:
```
  "minItems": 2,
  "maxItems": 2
```
^ Should also validate the length of each item in our array

Reply:
the positional specification does this